TestApp/TestApp/CustomViews/AtomicComponents/Molecules/VariableSizeLabelView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  VariableSizeLabelView.swift
+//  ECCBWallet
+//
+//  Created by Jonathan Bott on 11/1/19.
+//  Copyright © 2019 Jon Bott. All rights reserved.
+//
+
+import Foundation
+import DesignSystem
+
+public class VariableSizeLabelView: UIView {
+
+    private var smallNumbersDefaultedToZero = true
+
+    public enum Style: String {
+        
+        
+        case `default`,
+              numPadOutput,
+              numPadOutputSmall,
+              numPadOutputSmallDefault,
+              accentRequest
+
+        public var typography: Typography {
+            switch self {
+            case .numPadOutput:
+                return .x5large
```

Comment:
Now I see why the typographys are here but again i'd prefer anything going into the test app uses the default stuff.. just to keep it lean